openapi.yaml

```diff
@@ -4161,14 +4164,25 @@ components:
           items:
             $ref: '#/components/schemas/Organization'
           type: array
+        datasources:
```

Comment:
This is currently quite useless - github is always present and jira could be derived from the existing `jira` property, but:
- it's a good abstraction
- can be later expanded, with "deployments" to start